copter/source/docs/sprayer.rst

```diff
@@ -32,13 +32,9 @@ Enabling the Sprayer
 
 -  Connect your autopilot to the ground station (i.e. Mission Planner)
 -  Set the :ref:`SPRAY_ENABLE <SPRAY_ENABLE>` parameter to 1 and refresh parameters (the sprayer is not a commonly used feature so its other parameters are initially hidden)
--  Connect the pump to one of the autopilot's auxiliary PWM outputs (like Pixhawk AUX1) and set the appropriate SERVO*_FUNCTION  or RC*_FUNCTION to 22 (where "*" is the RC output number, i.e. if the pump is connected to a Pixhawk's AUX1, set :ref:`SERVO9_FUNCTION <SERVO9_FUNCTION>` to 22).
--  Connect the optional spinner to another auxiliary output port and set SERVO*_FUNCTION or RC*_FUNCTION to 23 (i.e. if using a Pixhawk's AUX2, set :ref:`SERVO10_FUNCTION <SERVO10_FUNCTION>` to 23)
--  To allow the pilot to turn on/off the sprayer set an auxiliary switch (i.e. CH7_OPT in versions prior to Copter-4.0 or an ``RCx_OPTIONS`` in Copter-4.0 or higher) to "15"
-
-.. note::
-
-   in Copter 3.5 (and higher) the RCx_FUNCTION parameters were renamed to SERVOx_FUNCTION to better separate their meaning from the RC input.
+-  Connect the pump to one of the autopilot's PWM outputs (like Pixhawk AUX1) and set the appropriate SERVO*_FUNCTION  or RC*_FUNCTION to 22 (where "*" is the RC output number, i.e. if the pump is connected to a Pixhawk's AUX1, set :ref:`SERVO9_FUNCTION <SERVO9_FUNCTION>` to 22).
```

Comment:
Still have a "Pixhawk" in this line.  I think we can just rename it to "AUX OUT1" or just remove it.

Reply:
I do not change Pixhawk if its used as an example rather than an exclusive